hack/setup-egress-ai-mock.sh

```
@@ -0,0 +1,195 @@
+      command: ["sleep", "infinity"]
+  restartPolicy: Never
+EOF
+
+info "Waiting for team-gold pod to be ready..."
+kubectl wait --timeout=2m -n "$EGRESS_TEST_NS" pod/team-gold --for=condition=Ready
+
+# ── Verify connectivity ──────────────────────────────────────────────
+info "Verifying AI mock connectivity through egress gateway..."
+
+info "Waiting for gateway address..."
+for attempt in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)